plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakePerTransactionMetastoreCache.java

```diff
@@ -35,10 +34,6 @@
 import io.trino.plugin.hive.metastore.thrift.ThriftMetastoreAuthenticationModule;
 import io.trino.plugin.hive.metastore.thrift.ThriftMetastoreClientFactory;
 import io.trino.plugin.hive.metastore.thrift.ThriftMetastoreConfig;
-import io.trino.spi.Plugin;
```

Comment:
commit message "Let TestingDeltaLakePlugin accept extension module" -> "Accept extension module in TestingDeltaLakePlugin"